lib/cl/externals/software/bordeaux-threads-v0.8.5/.travis.yml

```diff
@@ -0,0 +1,44 @@
+language: lisp
```

Comment:
If you checked in bordeaux-threads, it should not be part of the submission.  Instead, the build process should install it on demand in the same manner that the nodejs implementation calls npm to install packages during the build.